extensions/amp-date-picker/1.0/component.jss.js

```diff
@@ -0,0 +1,14 @@
+import {createUseStyles} from 'react-jss';
+
+// DO NOT SUBMIT: Example class used for styling
```

Comment:
I can remove this file if it's determined that this component doesn't need JSS, but I'm uncertain about this at the moment.

Reply:
We should remove this if we don't need any custom preact styles